src/formatDate.js

```diff
@@ -8,7 +8,51 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let day = 0;
+  let month = 0;
+  let year = 0;
```

Comment:
The variables 'day', 'month', and 'year' are initialized with 0, which is not used anywhere in the code. Instead, they are reassigned with the index positions of the corresponding date parts. Consider initializing them directly with the index positions.